examples/downstream-v0.sh

```diff
@@ -3,14 +3,18 @@
 # This script will run a local OLMv0 downstream operation, to allow someone to
 # debug a failure.
 #
-# Assumptions:
-# * The tooling directory is located in ${HOME}/git/openshift/operator-framework-tooling
-# * The v0 executable has been built in there via `make`
-#
 # Call with your github user name, if it's not the same as your current user name.
 #
+# Usage examples:
+#     downstream-v1.sh octocat "-drop-commits fca12345678" /Users/octocat/downstreamv0
+#     downstream-v1.sh octocat "" /Users/octocat/downstreamv0
+#     downstream-v1.sh octocat
 
 GITHUB_USER=${1:-${USER}}
+SYNC_ARGS=${2:-""}
+SYNC_DIR=${3:-$PWD}
```

Comment:
This while section becomes problematic when one wants to leave off an argument: i.e. you _have_ to include your GitHub user id if you want to specify any arguments.

Reply:
I'm wondering if it makes more sense to use variables beforehand?
e.g.
`GITHUB_USER=tmshort ./downstream-v0.sh`

Reply:
Or even to inform the script via explicit options instead of positional arguments.

Reply:
And this is how the scope got of the work got expanded :)

Switched to options.